exercises/http_collect/solution/solution_alt.js

```diff
@@ -0,0 +1,16 @@
+const http = require('http')
+let outputData = ''
+
+http.get(process.argv[2], function (response) {
```

Comment:
I think if we use es6 syntax for variables in the solution, should use es6 arrow functions too